perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1037,7 +1037,7 @@ sub enquiry : Chained('property') : Args(0) {
 
     my $category = $c->get_param('category');
     my $service = $c->get_param('service_id');
-    $c->detach('property_redirect') unless $category && $service && $c->stash->{services}{$service};
+    $c->detach('property_redirect') unless $category;
```

Comment:
Fine for now, but guess we might want to make this still check `$service` exists for Bromley where it is required.

Reply:
https://github.com/mysociety/societyworks/issues/4382